### PR TITLE
[Hotfix][Jdbc/CDC] Fix postgresql uuid type in jdbc read

### PR DESCRIPTION
**Pull request by hailin0**
### Purpose of this pull request

[Jdbc/CDC] Fix postgresql uuid type in jdbc read

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Added

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/splitter/JdbcSourceChunkSplitter.java [hunk 1]
```diff
@@ -54,12 +68,19 @@ Object[] queryMinMax(JdbcConnection jdbc, TableId tableId, String columnName)
...
-            JdbcConnection jdbc, TableId tableId, String columnName, Object excludedLowerBound)
+    default Object queryMin(
+            JdbcConnection jdbc, TableId tableId, Column column, Object excludedLowerBound)
+            throws SQLException {
+        return queryMin(jdbc, tableId, column.name(), excludedLowerBound);
+    }
+
+    @Deprecated
+    Object[] sampleDataFromColumn(
+            JdbcConnection jdbc, TableId tableId, String columnName, int samplingRate)
```
Comment (TaoZex) [inline seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/splitter/JdbcSourceChunkSplitter.java:83]: Add code comment
```suggestion
    /**
     * @deprecated instead by {@link this#sampleDataFromColumn(JdbcConnection, TableId, Column, samplingRate}
     */
    @Deprecated
    Object[] sampleDataFromColumn(
            JdbcConnection jdbc, TableId tableId, String columnName, int samplingRate)
```

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java [hunk 2]
```diff
@@ -405,4 +409,8 @@ default JdbcConnectionProvider getJdbcConnectionProvider(
             JdbcConnectionConfig jdbcConnectionConfig) {
         return new SimpleJdbcConnectionProvider(jdbcConnectionConfig);
     }
+
+    default String convertType(String columnName, String columnType) {
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java:420]: Any comment for this method? I can not get how to use it.

File: seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/splitter/JdbcSourceChunkSplitter.java (hunk 1 above)
Comment (hailin0) [reply seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/splitter/JdbcSourceChunkSplitter.java:83]: here
https://github.com/apache/seatunnel/pull/6684/files#diff-5fc13798b132b881e1464399fcbc8aa8febba01870c17cb32dc77496b3c8ada9R98

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java (hunk 2 above)
Comment (hailin0) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java:420]: added docs

**Review by TaoZex — APPROVED** [review]
+1

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by Carl-Zhou-CN — APPROVED** [review]
+1

**Pull request merged**